services-core/src/gas_price/gas_station.rs

```diff
@@ -63,12 +64,8 @@ impl GnosisSafeGasStation {
 
 #[async_trait::async_trait]
 impl GasPriceEstimating for GnosisSafeGasStation {
-    async fn estimate(&self) -> Result<U256> {
-        Ok(self.gas_prices().await?.fast)
-    }
-
     async fn estimate_with_limits(&self, _gas_limit: U256, _time_limit: Duration) -> Result<U256> {
-        self.estimate().await
+        Ok(self.gas_prices().await?.fast)
```

Comment:
Should we (separately) implement something similar here where fast, fastest etc. map to different target times?

Reply:
We probably will. I'm not exactly sure yet what it would be because the gnosis safe gas station is defined over gas price percentiles in previous blocks which isn't a direct indicator of time.